vllm/model_executor/models/bert.py

```diff
@@ -508,8 +496,33 @@ def _build_pooler(self, pooler_config: PoolerConfig) -> Pooler:
         })
 
 
-class BertForSequenceClassification(nn.Module, SupportsV0Only,
-                                    SupportsCrossEncoding, SupportsQuant):
+TOKEN_TYPE_SHIFT = 30
+
+
+def _encode_token_type_ids(input_ids: torch.tensor,
```

Comment:
This admittedly is strange, but the model inputs have to be persistent tensors and the model code itself can't have branches for it to work with cuda graphs.